src/renderer/css/global.css

```diff
@@ -1,24 +1,24 @@
 body {
-    user-select: none;
+	user-select: none;
 }
 
 /* disable dragging links */
 a {
-    -webkit-user-drag: none;
+	-webkit-user-drag: none;
 }
 
 .ui.modal {
-    user-select: none !important;
+	user-select: none !important;
 }
 
 /* fix FUI glitch */
 .ui.basic.modal .ui.form.inverted .ui.header {
-    color: white;
+	color: white;
 }
 
 /* font awesome right padding */
 .svg-inline--fa {
-    margin-right: 7px;
+	margin-right: 7px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5444487)